workspace/utilities/lib/meta-theme.xsl

```diff
@@ -2,8 +2,8 @@
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
 
 	<xsl:template name="master-theme-metas">
-		<xsl:if test="string-length($config/theme) != 0">
-			<meta name="theme-color" content="{$config/theme}" />
+		<xsl:if test="string-length($config/meta-theme) != 0">
```

Comment:
why not check/keep both ?

Reply:
the conventions changed for this. I would prefer if we stick with one. In that case, meta-theme